### PR TITLE
Release alluxio-enterprise 2.4.2-2.1.2-1.0 (automated commit)

### PR DESCRIPTION
**Pull request by bf8086**
Description:
Alluxio Enterprise Edition

Source URL: https://alluxio-dcos-dev.s3.amazonaws.com/autodelete7d/alluxio-enterprise/20200204-130546-KygnoR78HauvYfjY/stub-universe-alluxio-enterprise.json

Changes between revisions 300 => 400:
0 files added: []
0 files removed: []
3 files changed:

```
--- 300/marathon.json.mustache
+++ 400/marathon.json.mustache
@@ -21,10 +21,10 @@
   },
   {{/service.service-account-secret}}
   "env": {
-    "PACKAGE_BUILD_TIME_EPOCH_MS": "1576876153455",
-    "PACKAGE_BUILD_TIME_STR": "Fri Dec 20 2019 21:09:13 +0000",
+    "PACKAGE_BUILD_TIME_EPOCH_MS": "1580850348714",
+    "PACKAGE_BUILD_TIME_STR": "Tue Feb 04 2020 21:05:48 +0000",
     "PACKAGE_NAME": "alluxio-enterprise",
-    "PACKAGE_VERSION": "2.4.1-2.1.1-1.0",
+    "PACKAGE_VERSION": "2.4.2-2.1.2-1.0",

     "BOOTSTRAP_URI": "{{resource.assets.uris.bootstrap-zip}}",
     "EXECUTOR_URI": "{{resource.assets.uris.executor-zip}}",
@@ -48,7 +48,7 @@
     "SERVICE_USER": "{{service.user}}",
     "TASKCFG_ALL_ALLUXIO_DISTRIBUTION": "{{service.alluxio-distribution}}",
     "TASKCFG_ALL_ALLUXIO_LICENSE": "{{service.license}}",
-    "TASKCFG_ALL_ALLUXIO_VERSION": "2.1.1-1.0",
+    "TASKCFG_ALL_ALLUXIO_VERSION": "2.1.2-1.0",
     "TASKCFG_ALL_LOG_LEVEL": "{{service.log-level}}",
     "TASKCFG_ALL_SERVICE_NAME": "{{service.name}}",

--- 300/package.json
+++ 400/package.json
@@ -3,7 +3,7 @@
   "upgradesFrom": [],
   "downgradesTo": [],
   "name": "alluxio-enterprise",
-  "version": "2.4.1-2.1.1-1.0",
+  "version": "2.4.2-2.1.2-1.0",
   "maintainer": "info@alluxio.com",
   "description": "Alluxio, formerly Tachyon, is the world's first system that unifies disparate storage systems at memory speed. With Alluxio, enterprises can manage data efficiently, accelerate business analytics, and ease the adoption of hybrid cloud. \n\n Alluxio Enterprise Edition delivers operational benefits of a stable, high performance system with enterprise features and support for critical deployments. Please refer to https://alluxio.com/docs/enterprise/1.7/en/Alluxio-on-DCOS.html for a detailed operation guide. \n\n Release Notes: \n - Upgrade to version 2.1.0 of Alluxio",
   "selected": false,
--- 300/resource.json
+++ 400/resource.json
@@ -5,12 +5,12 @@
       "alluxio-agent-darwin_amd64": "https://alluxio-dcos-release.s3.amazonaws.com/agent/v1.5.0/darwin_amd64/alluxio-dcos",
       "alluxio-agent-linux_386": "https://alluxio-dcos-release.s3.amazonaws.com/agent/v1.5.0/linux_386/alluxio-dcos",
       "alluxio-agent-linux_amd64": "https://alluxio-dcos-release.s3.amazonaws.com/agent/v1.5.0/linux_amd64/alluxio-dcos",
-      "bootstrap-zip": "https://alluxio-dcos-release.s3.amazonaws.com/artifacts/2.4.1-2.1.1-1.0/bootstrap.zip",
-      "executor-zip": "https://alluxio-dcos-release.s3.amazonaws.com/artifacts/2.4.1-2.1.1-1.0/executor.zip",
+      "bootstrap-zip": "https://alluxio-dcos-release.s3.amazonaws.com/artifacts/2.4.2-2.1.2-1.0/bootstrap.zip",
+      "executor-zip": "https://alluxio-dcos-release.s3.amazonaws.com/artifacts/2.4.2-2.1.2-1.0/executor.zip",
       "java-tar-gz": "https://alluxio-dcos-release.s3.amazonaws.com/jdk/jdk1.8.0_144_jce_policy.tar.gz",
       "jre-tar-gz": "https://downloads.mesosphere.com/java/server-jre-8u162-linux-x64.tar.gz",
       "libmesos-bundle-tar-gz": "https://downloads.mesosphere.com/libmesos-bundle/libmesos-bundle-master-28f8827.tar.gz",
-      "scheduler-zip": "https://alluxio-dcos-release.s3.amazonaws.com/artifacts/2.4.1-2.1.1-1.0/alluxio-enterprise-scheduler.zip"
+      "scheduler-zip": "https://alluxio-dcos-release.s3.amazonaws.com/artifacts/2.4.2-2.1.2-1.0/alluxio-enterprise-scheduler.zip"
     }
   },
   "images": {
@@ -29,7 +29,7 @@
             }
           ],
           "kind": "executable",
-          "url": "https://alluxio-dcos-release.s3.amazonaws.com/artifacts/2.4.1-2.1.1-1.0/dcos-service-cli-darwin"
+          "url": "https://alluxio-dcos-release.s3.amazonaws.com/artifacts/2.4.2-2.1.2-1.0/dcos-service-cli-darwin"
         }
       },
       "linux": {
@@ -41,7 +41,7 @@
             }
           ],
           "kind": "executable",
-          "url": "https://alluxio-dcos-release.s3.amazonaws.com/artifacts/2.4.1-2.1.1-1.0/dcos-service-cli-linux"
+          "url": "https://alluxio-dcos-release.s3.amazonaws.com/artifacts/2.4.2-2.1.2-1.0/dcos-service-cli-linux"
         }
       },
       "windows": {
@@ -53,7 +53,7 @@
             }
           ],
           "kind": "executable",
-          "url": "https://alluxio-dcos-release.s3.amazonaws.com/artifacts/2.4.1-2.1.1-1.0/dcos-service-cli.exe"
+          "url": "https://alluxio-dcos-release.s3.amazonaws.com/artifacts/2.4.2-2.1.2-1.0/dcos-service-cli.exe"
         }
       }
     }
```

### CODE REVIEW

**Comment by bf8086** [comment]
@ryadav88 Please take a look.

File: repo/packages/A/alluxio-enterprise/400/package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "version": "2.4.2-2.1.2-1.0",
+  "maintainer": "info@alluxio.com",
+  "description": "Alluxio, formerly Tachyon, is the world's first system that unifies disparate storage systems at memory speed. With Alluxio, enterprises can manage data efficiently, accelerate business analytics, and ease the adoption of hybrid cloud. \n\n Alluxio Enterprise Edition delivers operational benefits of a stable, high performance system with enterprise features and support for critical deployments. Please refer to https://alluxio.com/docs/enterprise/1.7/en/Alluxio-on-DCOS.html for a detailed operation guide. \n\n Release Notes: \n - Upgrade to version 2.1.0 of Alluxio",
+  "selected": false,
+  "framework": true,
+  "tags": [
+    "enterprise",
+    "alluxio"
+  ],
+  "minDcosReleaseVersion": "1.10",
```
Comment (realmbgl) [inline repo/packages/A/alluxio-enterprise/400/package.json:15]: You may want to adjust the `minDcosReleaseVersion`.

Comment (bf8086) [reply repo/packages/A/alluxio-enterprise/400/package.json:15]: @realmbgl Please take a look at the new PR which updates the DC/OS release version - https://github.com/mesosphere/universe/pull/2502

**Comment by bf8086** [comment]
Closing in favor of an updated PR #2502.

**Pull request closed without merging**